doc/rest.md

```diff
@@ -438,6 +442,9 @@ JSON array of objects representing available maps with following structure:
   - **write**: Array of strings. Names of [users](./models.md#user) and [roles](./models.md#role) with [write access](./security.md#Authorization).
 - **bounding_box**: List of 4 floats. Bounding box coordinates [minx, miny, maxx, maxy] in EPSG:3857.
 
+Headers:
+- **X-Total-Count**: Total number of publications available from the request. Example `"247"`.
```

Comment:
Total number of publications available from the request, taking into account all filtering parameters except `limit` and `offset`. Example `"247"`.